.travis.yml

```diff
@@ -7,4 +7,4 @@ rust:
   - nightly
 
 script:
-  - cd utf8parse && cargo test
+  - cargo test && cd utf8parse && cargo test
```

Comment:
this should probably be split across two entries:

```yaml
script:
  - tests1
  - tests2
```

Reply:
I didn't even know you could do that! Done.